src/main/app/forms/models/address.ts

```diff
@@ -20,7 +21,7 @@ export class ValidationErrors {
   static readonly CITY_NOT_VALID: string = 'The city must be no longer than $constraint1 characters'
 
   static readonly POSTCODE_REQUIRED: string = 'Enter postcode'
-  static readonly POSTCODE_NOT_VALID: string = 'The postcode must be no longer than $constraint1 characters'
+  static readonly POSTCODE_NOT_VALID: string = 'Enter a valid postcode'
```

Comment:
'Enter a valid postcode

but

'Enter postcode' 

above. Shouldn't be there an "a" as well (or removed in both)?

Reply:
All error messages where user enter invalid value has `a`. For example,
```
Enter a valid town/city
Enter a valid year
Enter a valid amount owed
Enter a valid first payment date
```

Reply:
Discussed with Tarik and Marc, Both error messages are fine as it is.
`Enter a valid postcode` for invalid input and `Enter postcode` for empty.

Reply:
ok, thank you :)